layers/meta-tegra-support/recipes-test/tegra-tests/vpi1-tests/run-vpi1-tests.sh.in

```diff
@@ -0,0 +1,171 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021, OpenEmbedded for Tegra Project
+#
+# WARNING: make sure you have at least a couple of GiB of free
+# space availabe in the current working directory, as some of
+# these samples generate very large files.
+#
+# Some of these tests work better if you use jetson_clocks to
+# speed things up.
+
+SOC_FAMILY="@SOC_FAMILY@"
```

Comment:
You can get the SoC type  at runtime by reading `/sys/module/tegra_fuse/parameters/tegra_chip_id`.

Reply:
I was not sure about this. getting the SOC type at the RUNTIME ?
Is the `Xavier NX` and `Xavier AGX` have the same `tegra_chip_id` ? if so, I will update the test script to get SOC type at the RUNTIME.

Reply:
0x21 = Tegra210 (Nano, TX1)
0x18 = Tegra186 (TX2)
0x19 = Tegra194 (Xavier)

I don't remember off-hand whether the value read from sysfs is decimal or hex, and whether it's consistently one or the other, across the three SoC types.

Reply:
it is a decimal value from the sysfs. It should be OK 
I will apply updates and push the new changes.